types/three/src/core/BufferAttribute.d.ts

```diff
@@ -61,6 +61,8 @@ export class BufferAttribute {
     set needsUpdate(value: boolean);
 
     readonly isBufferAttribute: true;
+    readonly isInterleavedBufferAttribute: undefined;
+    readonly isGLBufferAttribute: undefined;
```

Comment:
if it's undefined, we don't have to explicitly say this I think.

Reply:
ah, I just saw the message you left in the previous PR. https://github.com/three-types/three-ts-types/pull/295#discussion_r1062246152

If it's the solution I feel like we have to set `isXXX` to every Three.js class, and I don't think it's the right way.
(e.g. saying `readonly isMesh: undefined` in `Matrix4` because someone might want to use that)
I believe people solve this kind of problems in other ways in the TypeScript realm.

Reply:
Sorry about the delayed response. I think you make a fair point there. It could indeed get unwieldy quite fast if we would need to cover all the fields in similar fashion. So I've now removed the explicit `undefined` field definitions and switched to using `instanceof` in the pointercontrols test.